Cargo.toml

```
@@ -28,3 +28,7 @@ members = [
     "trin-cli",
     "ethportal-peertest"
 ]
+
+[patch.crates-io]
+# Use temporaly discv5 version
+discv5 = { git = "https://github.com/ogenev/discv5.git", branch = "public_kbucket" }
```

Why use this instead of something like:
```suggestion
[dependencies.discv5]
branch = "public_kbucket"
git = "https://github.com/ogenev/discv5"
```

It seems like `patch.crates-io` requires also installing the patch crate and running `cargo patch`, based on: https://crates.io/crates/cargo-patch

---

Ah nice, thanks for the link